videotroncmdb.json

```diff
@@ -0,0 +1,290 @@
+{
+    "appid": "e9279bd9-6fbf-4aae-9982-83109d9e5423",
+    "name": "Videotron CMDB",
+    "description": "This app integrates with the Videotron CMDB tool to perform lookups",
+    "type": "information",
+    "product_vendor": "Videotron",
+    "logo": "logo_videotroncmdb.svg",
+    "logo_dark": "logo_videotroncmdb_dark.svg",
+    "product_name": "CMDB",
+    "python_version": "3",
+    "product_version_regex": ".*",
+    "publisher": "Splunk",
+    "license": "Copyright (c) 2020-2021 Splunk Inc.",
```

Comment:
Change the copyright year to `2021` as it will be released for the first time.

Reply:
Changed the copyright year to 2021.